### PR TITLE
VSFeat: Export Logic App

### PR DESCRIPTION
**Pull request by Eric-B-Wu**
Added Export Logic App

vscode_Export

Also just ran an xml formatter on the SVG icons.

### CODE REVIEW

**Review by rllyy97 — APPROVED** [review]

**Comment by Eric-B-Wu** [comment]
oops I already merged, i'll make these changes in a separate PR

File: apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+    localSettingsFile.Values[workflowLocationKey] = this.location;
+    localSettingsFile.Values[workflowManagementBaseURIKey] = 'https://management.azure.com/';
+
+    writeFileSync(`${this.targetDirectory}/parameters.json`, JSON.stringify(parametersFile, null, 4));
+    writeFileSync(`${this.targetDirectory}/local.settings.json`, JSON.stringify(localSettingsFile, null, 4));
+    this.addStatus(localize('done', 'Done.'));
+  }
+}
+
+export async function exportLogicApp(_context: IActionContext): Promise<void> {
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts:295]: NIT: if you don't need the `_context` maybe don't put it.

File: apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+  }
+
+  const options: vscode.WebviewOptions & vscode.WebviewPanelOptions = {
+    enableScripts: true,
+    retainContextWhenHidden: true,
+  };
+
+  const panel: vscode.WebviewPanel = vscode.window.createWebviewPanel('ExportLA', `${panelName}`, vscode.ViewColumn.Active, options);
+
+  const indexPath = join(ext.context.extensionPath, 'dist/designer/vs-code-react/index.html');
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts:327]: You can remove this chunk of code by using the `getWebViewHTML` function, just pass the `vs-code-react` folder name. Here it is an example of how to do it. 
`apps/vs-code-designer/src/app/commands/workflows/openOverview.ts:104`

File: apps/vs-code-designer/src/package.json [hunk 3]
```diff
@@ -470,6 +484,7 @@
     "onCommand:logicAppsExtension.deleteLogicApp",
     "onCommand:logicAppsExtension.openOverview",
     "onCommand:logicAppsExtension.refresh",
+    "onCommand:azureLogicAppsStandard.exportLogicApp",
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/package.json:487]: "onCommand:logicAppsExtension.exportLogicApp" instead

**Pull request merged**